doc/workshops/tutorial/content/examples/tutorial3-fenix.md

```diff
@@ -0,0 +1,99 @@
+# Tutorial 3 - Kinetic Implementation
+
+!media media/FENIX_logo_blue.png
+       style=width:50%;display:block;margin-top:1.5em;margin-left:auto;margin-right:auto;box-shadow:none;
```

Comment:
This is so big that on my ultrawide display, the link doesn't even show up. I would drop the size of this.

Reply:
I have adjusted this to 30% let me know if that is a good size.